python/HeaderService/hutils.py

```diff
@@ -140,7 +143,6 @@ def md5Checksum(filePath, blocksize=1024*512):
 
 
 def get_obsnite(date=None, thresh_hour=14, format='{year}{month:02d}{day:02d}'):
```

Comment:
Is this function used any more now that DAYOBS is being extracted from OBSID?

Reply:
Not anymore, your comment implies that it is okay to get rid of it. Therefore that what I'll do.